integration-tests/aws-itests/pom.xml

```diff
@@ -27,15 +27,28 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>camel-quarkus-integration-test-aws-sns</artifactId>
-    <name>Camel Quarkus :: Integration Tests :: AWS SNS</name>
+    <artifactId>camel-quarkus-integration-test-aws-itests</artifactId>
```

Comment:
Both the name of the directory `aws-itests`  and the name of the artifactId `camel-quarkus-integration-test-aws-itests` does not match the naming convention used in all other itests. Would you mind changing the dir to `aws` and artifactId to `camel-quarkus-integration-test-aws`?